packages/@tinacms/fields/src/Wysiwyg/state/plugins/Menu/markControl.tsx

```diff
@@ -98,8 +98,6 @@ export function markControl({
       }
 
       toggleMark(this.markType(mark), defaultAttrs)(state, dispatch)
```

Comment:
Following change is to fix focus in link modal.

Reply:
Successfully tested locally